pkg/gardenlet/controller/shoot/shoot/reconciler.go

```diff
@@ -148,29 +148,12 @@ func (r *Reconciler) reconcileShoot(ctx context.Context, log logr.Logger, shoot
 		return reconcile.Result{}, errorsutils.WithSuppressed(syncErr, updateErr)
 	}
 
+	reportMetrics(shoot, operationType, r.Clock.Now().UTC().Sub(shoot.CreationTimestamp.Time).Seconds())
```

Comment:
We could use the creation timestamp in UTC – just to be sure.
```suggestion
	reportMetrics(shoot, operationType, r.Clock.Now().UTC().Sub(shoot.CreationTimestamp.UTC()).Seconds())
```